manifests/Fighter Factory.yaml

```diff
@@ -0,0 +1,14 @@
+name: Fighter Factory
+authors: 1010todd
+homepage: https://github.com/1010todd/Fighter-Factory
+license: GPL-3.0-or-later
+version: f3bc8e4ccc9ac9fba58e10f136acbf0d58fc55bc
+shortDescription: Add outfit to produce replacement fighters. Updated to latest stable.
+description: Add outfit to produce replacement fighters. To use, after installing the Fighter Assembler outfit depart and land once then check the job board to record the current number of fighters. Now if the number of fighter ever drop below the recorded number new one will be made to replace it as long as supply is available. Will be updated to latest stable release.
+url: https://github.com/1010todd/Fighter-Factory/archive/f3bc8e4ccc9ac9fba58e10f136acbf0d58fc55bc.zip
+iconUrl: https://github.com/1010todd/Fighter-Factory/raw/f3bc8e4ccc9ac9fba58e10f136acbf0d58fc55bc/icon.png
```

Comment:
the icon's a lie.

Reply:
Added the icon now: https://github.com/1010todd/Fighter-Factory/commit/9db8b336882c4b613316c6a4260f56af7ec885cc